content/joysticks/winwing/winwing-cdu/troubleshooting/_index.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Troubleshooting
+weight: 30
+---
```

Comment:
This should be a level higher in the hierarchy and be a dedicated page for all WINWING troublehshooting, not just CDU.

Reply:
I'm not sure, I expect very specific CDU issues, since that has all the Python stuff and the auto-load and so on.